.github/workflows/documentation-creation.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Doxygen Action
+
+# Controls when the action will run. Triggers the workflow on push or pull request
```

Comment:
comment is now outdated

Reply:
Have removed the comments, you right that they no more matches.